sparse_strips/vello_common/src/coarse.rs

```diff
@@ -333,7 +333,7 @@ impl Wide {
         let mut cur_wtile_y = clip_bbox.y0();
 
         // Process strips to determine the clipping state for each wide tile
-        for i in 0..n_strips - 1 {
+        for i in 0..n_strips.saturating_sub(1) {
```

Comment:
There're two other cases above where `n_strips - 1` is used when calculating bbox. I know there’s a precondition that returns an empty bbox when `n_strips <= 1`, but it might still be worth keeping the approach consistent here.